apps/s_server.c

```diff
@@ -2202,22 +2202,25 @@ static int sv_body(int s, int stype, int prot, unsigned char *context)
             BIO_printf(bio_err, "Turned on non blocking io\n");
     }
 
+    con = SSL_new(ctx);
     if (con == NULL) {
-        con = SSL_new(ctx);
+	ret = -1;
+	goto err;
```

Comment:
Formatting nit, replace tabs with spaces.

Reply:
@dot-asm Thank you for your review. I replaced tabs with spaces.